core/src/main/scala/scalaz/Cofoldable1.scala

```diff
@@ -0,0 +1,29 @@
+package scalaz
+
+trait Cofoldable1[F[_], A] {
```

Comment:
why not the more general definition:
```scala
trait Cofoldable1[F, A] {
  def build1[B](b: B)(f: B => (A, Option[B])): F
}
```
? 
type-inference?
(also it would allows `Reducer[C, M] extends Cofoldable1[M, C]`)

Reply:
Let me think about that one...